src/components/Navigator/NavigatorCard.vue

```diff
@@ -932,6 +932,8 @@ export default {
      * that points to another technology.
      */
     handleNavigationChange(uid) {
+      // force-close the navigator on mobile
+      this.$emit('close');
```

Comment:
This seems to also affect the desktop navigator.

Reply:
yeah, fixed.